README.md

```diff
@@ -475,6 +476,19 @@ These behave exactly like VueJs's transition directives, except they have differ
 
 ---
 
+### `x-ignore`
+**Example:**
+```html
+<div x-ignore="true">
+    <div x-data="{foo:'bar'}">
+        <span x-text="foo">"foo" never gets bind by alpine</span>
+    </div>
+</div>
+```
+`x-ignore` can be used to stop DOM walking and ignoring any components within the element.
```

Comment:
`x-ignore` can be used to mark an element's children as "to be ignored" by Alpine. This means that Alpine will not walk the DOM tree any further and will not initialise any components within the `x-ignore`-ed element.

Depending on whether we implement the `<div x-data x-ignore>` behaviour we might have to amend this: `mark an element's children` to `mark an element`

Reply:
-  Done. Thanks for the better description.